examples/website/src/shared/utility.ts

```diff
@@ -125,6 +125,16 @@ export function transformCssStringToObject(styleCustom: string): object {
   return styleRules.reduce((previousValue: object, currentValue: string) => {
     const [key, value] = currentValue.split(':')
     if (key && value) {
+      const index = key.indexOf('-')
+      if (index !== -1) {
+        let camelCaseKey = key.substring(index)
+        camelCaseKey = camelCaseKey.slice(1).replace(camelCaseKey[1], camelCaseKey[1].toUpperCase())
+
+        return Object.assign(previousValue, {
+          [(key.substring(0, index) + camelCaseKey).trim()]: value.trim()
+        })
+      }
+
```

Comment:
This seems to cover only single dash cases - I think there might be more.

Wdyt about using something more generic for the `key`?
```
key.toLowerCase().replace(/\-(.)/gm, ($0, $1) => $1.toUpperCase())
```

Reply:
@tomaszdurka I'm not sure I understand the second half of this, but it indeed works.